workflows/sock-shop-demo/usingCMDProbe/workflow.yaml

```diff
@@ -278,6 +270,90 @@ spec:
           ]
 
+    - name: install-pod-network-loss
+      inputs:
+        artifacts:
+          - name: install-pod-network-loss
+            path: /tmp/pod-network-loss.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                description:
+                  message: |
+                    Injects network packet loss on pods belonging to an app deployment
+                kind: ChaosExperiment
+                metadata:
+                  name: pod-network-loss
+                spec:
+                  definition:
+                    scope: Namespaced
+                    permissions:
+                      - apiGroups:
+                          - ""
+                          - "batch"
+                          - "litmuschaos.io"
+                        resources:
+                          - "jobs"
+                          - "pods"
+                          - "pods/log"
+                          - "events"
+                          - "chaosengines"
+                          - "chaosexperiments"
+                          - "chaosresults"
+                        verbs:
+                          - "create"
+                          - "list"
+                          - "get"
+                          - "patch"
+                          - "update"
+                          - "delete"
+                          - "deletecollection"
+                    image: "litmuschaos/go-runner:latest"
+                    imagePullPolicy: Always
+                    args:
+                    - -c
+                    - ./experiments -name pod-network-loss
+                    command:
+                    - /bin/bash
+                    
+                    env:
+                    - name: TARGET_CONTAINER
+                      value: ''
+                    - name: LIB_IMAGE
+                      value: 'litmuschaos/go-runner:latest' 
+                    - name: NETWORK_INTERFACE
+                      value: 'eth0'
+                    - name: TC_IMAGE
+                      value: 'gaiadocker/iproute2'
+                    - name: NETWORK_PACKET_LOSS_PERCENTAGE
+                      value: '100' #in PERCENTAGE
+                    - name: TOTAL_CHAOS_DURATION
+                      value: '60' # in seconds
+                    - name: RAMP_TIME
+                      value: ''
+                    - name: LIB
+                      value: 'litmus'
+                    - name: PODS_AFFECTED_PERC
+                      value: ''
+                    - name: TARGET_POD
+                      value: ''
+                    - name: CONTAINER_RUNTIME
+                      value: 'docker'
+                    - name: TARGET_IPS
+                      value: ''
+                    - name: TARGET_HOSTS
+                      value: ''
+                    - name: SOCKET_PATH
+                      value: '/run/containerd/containerd.sock'
```

Comment:
As container Runtime is marked as docker, Can you please add docker socket path here?
sync all the env [refer](https://github.com/litmuschaos/chaos-charts/blob/master/charts/generic/pod-network-loss/experiment.yaml)